packages/firebase_auth/firebase_auth_web/lib/firebase_auth_web.dart

```diff
@@ -402,7 +402,14 @@ class FirebaseAuthWeb extends FirebaseAuthPlatform {
       // origin as a single string
       delegate.useAuthEmulator('http://$host:$port');
     } catch (e) {
-      throw getFirebaseAuthException(e);
+      final String code = (e as dynamic).code;
+      // this catches Firebase Error from web that occurs after hot reloading & hot restarting
+      // ignore: avoid_print
+      print(
+          "Web auth emulator throws exception on hot restart/reload (Web JS SDK thinks you're calling emulator multiple times). We're hiding the exception for your convenience.");
+      if (code != 'auth/emulator-config-failed') {
+        throw getFirebaseAuthException(e);
```

Comment:
We're losing the stacktrace here. The exception thrown should reuse the stacktrace

-> Error.throwWithStacktrace

Reply:
@rrousselGit  That makes sense. I guess this would be a separate PR for updating web so it makes use of `_flutterfire_internals`. Can open an issue for it?